docs/players/Installation_iOS.md

```diff
@@ -10,13 +10,18 @@ You can run VCMI on iOS 12.0 and later, all devices are supported. If you wish t
 To run on a non-jailbroken device you need to sign the IPA file, you
 have the following options:
 
-- (Easiest way) [AltStore](https://altstore.io/) or [Sideloadly](https://sideloadly.io/) - can be installed on Windows or macOS, don't require dealing with signing on your own 
-- if you're on iOS 14.0-15.4.1, you can try <https://github.com/opa334/TrollStore>
+- (Easiest way) [AltStore Classic](https://altstore.io/) or [Sideloadly](https://sideloadly.io/) - can be installed on Windows or macOS, doesn't require dealing with signing on your own 
+- if you're on iOS 14.0-15.4.1, you can try <https://github.com/opa334/TrollStore>. If you’re using this, no need to use any of the tools below. 
```

Comment:
well, I thought that "the following options" (as well as using a _bulleted_ list) before the list makes it clear that all these are exclusive :)

Reply:
I found myself wishing it was clear an explicit, because when I see a bulleted list it usually means you need to do all the bullet tasks to complete the main one.

Alternatively could maybe separate the easiest and recommended path first with Altstore all the way to launching the game, then provide all other alternatives below

Reply:
> Alternatively could maybe separate the easiest and recommended path first with Altstore all the way to launching the game, then provide all other alternatives below

sounds good. We can have 2 sectionsL AltStore as the easiest / recommended and "other ways"